meinberlin/apps/budgeting/rules.py

```diff
@@ -45,7 +46,7 @@
 
 rules.add_perm(
     "meinberlin_budgeting.moderate_proposal",
-    module_predicates.is_context_moderator | module_predicates.is_context_initiator,
+    is_superuser | module_predicates.is_allowed_moderate_project,
```

Comment:
But the ```is_allowed_moderate_project``` already includes the superusers, right?

Reply:
How did I not see that?